libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -405,6 +411,8 @@ class AC_PosControl
     // output from controller
     float       _roll_target;           // desired roll angle in centi-degrees calculated by position controller
     float       _pitch_target;          // desired roll pitch in centi-degrees calculated by position controller
+    float       _heading;               // desired heading in centi-degrees calculated by position controller
```

Comment:
maybe these variables should be renamed to _yaw and _yaw_rate?  if they're in centidegrees then probably yaw_cd and yaw_rate_cds.

Reply:
The reason I specified this as heading instead of yaw is because I don't believe yaw is the correct term. The Yaw axis specifies the rotation about the body frame Z axis and the yaw angle is the euler representation of heading. I felt that given our move away from euler angles the term heading was more appropriate and correct.

Reply:
Renamed to keep it consistent with the existing code.